solution4.py

```diff
@@ -0,0 +1,18 @@
+bcainfo="""BCA course information:
+Bachelor of computer Application.
+It is a three years undergraduate course."""
+
+with open('BCA.txt','w') as bca:
+    bca.write(bcainfo)
+
+rows={'A':0,'B':0,'C':0}
+
+with open('BCA.txt','r') as intro_file:
+    lines=intro_file.readlines()
+    for line in lines:
+            f_char=line.strip()[0]
+            if f_char in rows:
+                rows[f_char]+=1

```

Comment:
explain this for loop. how it work.

Reply:
for line in lines: This is a loop that iterates over each element in the lines collection. Here, the code first calls the strip() method on the line to remove any leading or trailing spaces or newlines).